Corona-Warn-App/src/test/java/de/rki/coronawarnapp/risk/RiskLevelsTest.kt

```diff
@@ -1,16 +1,43 @@
 package de.rki.coronawarnapp.risk
 
 import com.google.android.gms.nearby.exposurenotification.ExposureSummary
+import de.rki.coronawarnapp.appconfig.AppConfigProvider
 import de.rki.coronawarnapp.server.protocols.ApplicationConfigurationOuterClass
+import io.kotest.matchers.shouldBe
+import io.mockk.MockKAnnotations
+import io.mockk.impl.annotations.MockK
 import junit.framework.TestCase.assertEquals
+import org.junit.Before
 import org.junit.Test
 
-class RiskLevelCalculationTest {
+class RiskLevelsTest {
+
+    @MockK lateinit var appConfigProvider: AppConfigProvider
+    private lateinit var riskLevels: DefaultRiskLevels
+
+    @Before
+    fun setUp() {
+        MockKAnnotations.init(this)
+        riskLevels = DefaultRiskLevels(appConfigProvider)
+    }
+
+    @Test
+    fun `is within defined level threshold`() {
+        // positive
```

Comment:
I'd prefer separate test cases, so it's obvious by the test case name which scenario failed, instead of having to look at the line number.